types/fabric/fabric-impl.d.ts

```diff
@@ -4480,3 +4481,50 @@ interface IUtil extends IUtilAnimation, IUtilArc, IObservable<IUtil>, IUtilDomEv
 	object: IUtilObject;
 	string: IUtilString;
 }
+
+export interface FilterBackend {
+	resources: Object;
```

Comment:
`Object` refers to a fabric Object, but from what I can tell [`resources`](https://github.com/kangax/fabric.js/blob/1fa3fdae31d2ee9a1c7c4bebb975d66b3171b5e9/src/filters/2d_backend.class.js#L26) is a plain Javascript object map. If so, use an indexed type like `{ [key: string]: HTMLCanvasElement }`.